apis/projectcontour/v1/httpproxy.go

```diff
@@ -146,6 +149,26 @@ type TLS struct {
 	EnableFallbackCertificate bool `json:"enableFallbackCertificate,omitempty"`
 }
 
+// CorsPolicy allows setting de CORS policy
+type CorsPolicy struct {
```

Comment:
I think that for any fields that are optional, we want to add `omitempty` to the JSON tag.